src/main/java/uk/ac/ic/wlgitbridge/bridge/util/CastUtil.java

```diff
@@ -0,0 +1,18 @@
+package uk.ac.ic.wlgitbridge.bridge.util;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * Created by winston on 01/07/2017.
+ */
+public class CastUtil {
+
+    public static int assumeInt(long l) {
```

Comment:
Even in programmer fonts I sometimes have to look twice to be sure an `l` isn't a `1`.